dasharo-performance/boot-time-measure.robot

```diff
@@ -26,7 +26,7 @@ Suite Teardown      Run Keyword
 
 
 *** Variables ***
-${ITERATIONS}=      5
+${ITERATIONS}=      6
```

Comment:
I would leave it at 5.  Why? Because it is the desired amount of iterations you want to measure. It should be independent of test workarounds.

Reply:
let's keep it tight then: https://github.com/Dasharo/open-source-firmware-validation/pull/538/commits/0c0b1a1df098656fab25619c4ef6a3d4f4cbec20